docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -0,0 +1,363 @@
+# Summary Collection
+
+The core data structure that tracks summary attempts and corresponding results by monitoring the op log.
+
+## SummaryAckWithoutOp
+
+> Error
+
+It means that a summary ack was observed without a corresponding summary op. We only raise this event if the missing summary op's sequence number >= the initial sequence number which we loaded from.
+
+Potential causes are that a summary op was nacked then acked, double-acked, or the `summarySequenceNumber` is invalid. All cases should be recoverable, but still indicate bad behavior.
+
+- `sequenceNumber` - sequence number of the observed summary ack op.
```

Comment:
It's not obvious that these are the fields in the event. This should be explicitly mentioned for clarity.